lcls_naming_tool/tests/test_cli.py

```diff
@@ -0,0 +1,86 @@
+# Unit tests for the LCLS Naming Tool script
+
+import pytest
+import json
+from pathlib import Path
+from lcls_naming_tool import display_version, load_taxons, functional_component_is_valid, fungible_is_valid, constituent_component_is_valid, increment_is_valid, starts_alphanumeric, validate
+
+
+@pytest.fixture
+def valid_name():
+    return ['MR2K4', 'KBO', 'PIP', '01', 'PUMPSIZE']
+
+
+@pytest.fixture
+def invalid_name():
+    return ['ABC12345M9', 'ABC', 'NBC', '7', '_PCTRLSPRBCK']
+
+
+def test_display_version(capsys):
+
+    display_version()
+    captured = capsys.readouterr()
+    assert str(captured.out).strip() == 'v1.0.0'
```

Comment:
This test will start to fail once we update the version number

Reply:
Updated to use `os.popen()` to get output from `git describe --tags`.